test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 bandit==1.7.2
 build>=0.7.0
-black==22.1.0
+black>=22.3.0
```

Comment:
There's a couple of other test-requirements that are pinned to out-of-date versions that I could update as well, but they don't seem to help/hurt anything so I'll leave them out rather than clutter things.